chapters/3-measurement.tex

```diff
@@ -37,125 +39,209 @@
-as well as the project folder where 
-the raw original data and codebooks are stored
-and where the back-ups for the each raw dataset are stored.
-
-Some of the characteristics in your master datasets and your data map
-should be filled in during the planning stage,
-but both of them should be active resources 
-that are updated as your project evolves.
-Finally, your master data should not include any unlabeled missing values. 
-If the information is missing for one unit, 
-then the reason should always be indicated with a code.
-An example for such reason could be that a unit was not included in the treatment assignment
-as it was not sampled in the first place,
-or was not located in the data collection at a given round.
-
-\subsection{Defining study comparisons using master data}
-
-Your research design will determine what statistical comparisons you need
-to estimate in your analytical work.
+Your data map must indicate which ID variables 
+can be used -- and how -- when merging datasets.
+The data linkage table is also a great place to list other types of metadata,
+such as the source of your data, its backup locations, and so on.
+
+Second, you should then create one \textbf{master dataset} 
+for each unit of observation
+used in any significant research activity.
+Examples of such activities are data collection, data analysis, 
+sampling, and treatment assignment.
+The master dataset should include and be the authoritative source of
+all \textbf{research variables}\sidenote{
+	\textbf{Research variables:} Research data that identifies observations
+	and maps research design information to those observations.
+	Research variables are time-invariant and 
+	often, but not always, controlled by the research team.
+	Examples include
+	ID variables, sampling status, treatment status, and treatment uptake.} 
+but not include any \textbf{measurement variables}\sidenote{
+	\textbf{Measurement variables:} Data that 
+	corresponds to direct observations of the real world, 
+	recorded sentiments of the subjects of the research 
+	or any other aspect your project is studying. 
+	Research variables are not controlled by the research team
+	and often vary over time.
+	Examples include characteristics of the research subject, 
+	outcome variables, input variables among many others.}.
+Research variables and measurement variables 
+often come from the same source,
+but should not be stored in the same way.
+For example, if you are shared administrative data that both includes 
+information on eligibility to be included in the study (research variable) 
+and information on outcome on the topic of your study (measurement variable) 
+you should first decide which variables are research variables, 
+remove them during the data cleaning (see Chapter 6) 
+and instead store them in your master dataset. 
+It is common that you will have to update 
+your master datasets throughout your project.
```

Comment:
This example makes sense to me, but we should discuss how to phrase it. Perhaps we can just indicate that they should be separated during data cleaning without attempting to describe a step-by-step process, and then do that in ch 6?

Reply:
research variables in master dataset, measurement variables cleaned dataset

Reply:
done in b6f584c8998d0e1052aa1d5ab137a10a2e2122cc